packages/ember-glimmer/lib/renderer.js

```diff
@@ -137,6 +138,7 @@ class Renderer {
       controller,
       targetObject: controller,
       outletState: view.toReference(),
+      rootOutletState: view.toReference(),
```

Comment:
I actually don't know if `rootOutletState` is needed. It's not used anywhere in this PR.